test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -175,6 +177,24 @@ export default class PerfCompare extends Component {
         })
     }
 
+    async getJenkinsRawData ( resBenchmarkBuildJson ) {
```

Comment:
It would be good to rename this function to `getChildrenRawData` since it only gets data for children. Also please add a line before each new function to say what it does.

Reply:
We could change `resBenchmarkBuildJson` to `buildJson`.